.overcommit.yml

```diff
@@ -0,0 +1,4 @@
+PreCommit:
```

Comment:
I prefer pre-commit because it avoids "fixing rubocop" commits in the history. We'll see if it gets too annoying though.